### PR TITLE
Adding integration test for cleanup published exposures

### PR DESCRIPTION
**Pull request by chaodaiG**
This is the first half of #640, creating an integration test for testing cleanup scenario of published exposures, testing exposure DB cleanup. 

It's planned to have a following PR testing the scenario of export cleanup, if needed I can add it in this PR as well

### CODE REVIEW

File: internal/integration/publish_test.go [hunk 1]
```diff
@@ -45,74 +48,76 @@ func TestPublish(t *testing.T) {
...
 
-		// TODO: hook up verification, and disable
-		BypassHealthAuthorityVerification: true,
+	// Create a signature info.
+	createSignatureInfo(ctx, db, exportPeriod, t)
+
+	firstPayload := newPayloads(3)
+	enClient.PublishKeys(t, firstPayload)
+	assertExposures(t, ctx, db, 3)
+	firstBatchKeys := sets.NewString()
```
Comment (sethvargo) [inline internal/integration/publish_test.go:65]: I really don't want to pull in apimachinery for this. Can we just use a map or string slice?

Comment (chaodaiG) [reply internal/integration/publish_test.go:65]: Sure, I can do that

Comment (chaodaiG) [reply internal/integration/publish_test.go]: Done

**Comment by sethvargo** [comment]
@chaodaiG looks like some legit failures for unused functions and stuff. You can run `make staticcheck` locally too

**Comment by chaodaiG** [comment]
> @chaodaiG looks like some legit failures for unused functions and stuff. You can run `make staticcheck` locally too

This is really cool! It caught my preparation for my next PR, deleted the unused function and will checkin in next PR. Need another lgtm I guess

**Comment by chaodaiG** [comment]
Needs another /lgtm

**Pull request merged**